sandbox/settings.py

```diff
@@ -126,9 +126,19 @@ class Base(Configuration):
     STATIC_ROOT = os.path.join(DATA_DIR, "static")
     MEDIA_URL = "/media/"
     MEDIA_ROOT = os.path.join(DATA_DIR, "media")
-
     STATICFILES_DIRS = (MACHINA_MAIN_STATIC_DIR,)
 
+    # AWS
+    AWS_ACCESS_KEY_ID = values.SecretValue()
+    AWS_S3_CUSTOM_DOMAIN = values.Value()
+    AWS_S3_REGION_NAME = values.Value()
+    AWS_S3_URL_PROTOCOL = values.Value("https")
+    AWS_SECRET_ACCESS_KEY = values.SecretValue()
+    AWS_STORAGE_BUCKET_NAME = values.Value()
+    AWS_DEFAULT_ACL = "public-read"
+    AWS_QUERYSTRING_AUTH = False
+    DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
```

Comment:
For the development of this feature, of course you had to use this.

But I agree with @jbpenrath, for newcomers wanting to test / contribute on Ashley, it would be more convenient to use the default storage engine. I see no reason to force them to use S3. My suggestion is to: 
 - use `values.Value("storages.backends.s3boto3.S3Boto3Storage")` in the Base settings
 - use `values.Value("django.core.files.storage.FileSystemStorage")` in the Dev settings. This would allow developers to override this value with an environment variable to work on S3 upload if necessary.